src/content/en/2019/a11y.md

```diff
@@ -5,3 +5,270 @@ title: Accessibility
+
+# Conclusion
+
+**P**eople legally defined as having a disability are not the only ones with accessibility needs. For example, anyone who has suffered a temporary wrist injury has experienced the difficulty of clicking small tap targets. Eyesight often diminishes with age, making text written in small fonts challenging to read. Finger dexterity is also not the same across age demographics, making tapping interactive controls or swiping through content on mobile websites more difficult for a sizable percentage of users.
+
+Similarly, assistive software is not only geared towards people with disabilities. For instance, drivers would benefit from a screen reading functionality that, while they keep their eyes on the road, reads long pieces of text like news stories aloud. The recent popularity of voice assistance, both on mobile devices and in the home, has demonstrated that controlling a computing device using voice commands is desirable, and at times essential, for a large number of users. Voice commands like these used to only be an accessibility feature, but is now turning into a mainstream product.
+
+Once a website is built, it’s often hard to retrofit accessibility on top of existing site structures and widgets. Accessibility isn’t something that can be easily sprinkled on afterwards, but needs to be part of the design and implementation process. Unfortunately, either through a lack of awareness, or easy to use testing tools, many developers are not familiar with the needs of all their users and the requirements of the assistive software they use.
+
+While not conclusive, our results indicate that the use of accessibility standards like ARIA, and good accessibility authoring practices (e.g., the use of alt text) are found on a *sizable, but not substantial,* portion of the web. On the surface this is encouraging, but we suspect many of these positive trends are due to the popularity of certain UI frameworks. On one hand this is disappointing because web developers cannot simply rely on UI framework to "infect" their sites with accessibility support. On the other hand though, it’s encouraging to see how large of an effect UI frameworks could have on the accessibility of the Web.
+
+The next frontier, in our opinion, is making widgets that are available through UI frameworks more accessible. Since many complex widgets used in the wild (e.g., calendar pickers) are sourced from a UI library, it would be great for these widgets to be accessible out of the box. What we hope are hoping for, is when we collect our results next time, the usage of more complex ARIA roles is on the rise—signifying more complex widgets have also been made accessible.
```

Comment:
Made some edits and added a final sentence.

The next frontier, in our opinion, is making widgets that are available through UI frameworks more accessible. Since many complex widgets used in the wild (e.g., calendar pickers) are sourced from a UI library, it would be great for these widgets to be accessible out of the box. What we hope are hoping for is, when we collect our results next time, that the usage of more complex ARIA roles is on the rise—signifying more complex widgets have also been made accessible. In addition, we hope to see a higher number of accessible media, like images and video, so that all users can enjoy the richness of the web.